src/sqarql-query.js

```diff
@@ -15,7 +15,14 @@ async function sparqlQuery(query) {
 			return json;
 		}
 	} catch(error) {
-		throw ['Fetch Error :-S', error];
+		const delay = Math.floor(Math.random() * 1000)
+		console.warn(`Sparql Query failed, retring in ${delay / 1000} seconds`)
+		console.log(error)
+
+		await new Promise(resolve => setTimeout(resolve, delay));
+
+		console.log('retring…');
+		return await sparqlQuery(query)
```

Comment:
this can potentially lead to an infinite recursion, so you want some form of max retry count.
Also it's usually a good idea to do the subsequent retries with growing delay (some kind of exponential decay usually works well, e.g. mutliply the originally chosen delay by 2 each time)

Reply:
I went for 5 attempts 🤷